docs/Gemfile

```diff
@@ -0,0 +1,9 @@
+source "https://rubygems.org"
+
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
```

Comment:
not sure if this remains same ? can you please explain me how does this line works.

Reply:
Yeah this has to remain the same , 
So basically this is one of the packages that is needed to build github pages in local ,